pkg/uotel/config.go

```
@@ -0,0 +1,513 @@
+	if tlsCertPath != "" && tlsCert != "" {
+		return nil, fmt.Errorf("tlsCertPath and tlsCert are mutually exclusive, only one should be provided")
+	}
+
+	// If both are empty, use the system certificate pool
+	if tlsCertPath == "" && tlsCert == "" {
+		zap.L().Debug("otel: no certificate provided, using system certificate pool")
+		systemPool, err := x509.SystemCertPool()
+		if err != nil {
+			return nil, fmt.Errorf("failed to load system certificate pool: %w", err)
+		}
+		return &tls.Config{
```

golangci-lint (1.64.8)

[high] G402: TLS MinVersion too low.

(gosec)